api/router/router.go

```diff
@@ -0,0 +1,19 @@
+package router
+
+import (
+	c "api/controller"
+
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
+)
+
+var Resp = render.New()
+
+func NewRouter() *mux.Router {
+	router := mux.NewRouter().StrictSlash(true)
+
+	router.HandleFunc("/api", c.HelloWorld)
+	router.HandleFunc("/api/import", c.ReadCSV)
```

Comment:
Remember that one of your routes should receive an ID to filter out the info you want to get

Reply:
Also if you're working with contestants. You would like to name your endpoint `/api/contestants/{id}`, or if you want to get all of them `/api/contestants`